src/index.js

```diff
@@ -203,9 +205,10 @@ class ReactNativeModal extends Component {
         return true;
       },
       onPanResponderMove: (evt, gestureState) => {
+        const { deviceWidth } = this.props;
```

Comment:
I would change this to:
`const deviceWidth = this.props.deviceWidth || this.state.deviceWidth`.